storage/file_test.go

```diff
@@ -37,7 +37,28 @@ func TestFile_GetPathInZip(t *testing.T) {
 	for _, test := range tests {
 		file.FileName = test.fileName
 		file.Folder = test.folder
-		assert.Equal(t, test.want, file.GetPathInZip())
+		assert.Equal(t, test.want, file.GetRelativePath())
+	}
+}
+
+func TestFile_GetFileNameAndExtension(t *testing.T) {
+	tests := []struct {
+	    scenario        string
+		filename        string
+		wantFilename    string
+		wantExtension   string
```

Comment:
Indentation gone a bit wierd here. Mix of tabs and spaces?

Reply:
yeah, I'm thinking of using this tool to fix indentations throughout the codebase, but probably best done in a separate PR, do you agree? - https://blog.golang.org/gofmt